thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/ThirdEyeEventsPipeline.java

```diff
@@ -48,12 +48,14 @@
 
   private static final String PROP_STRATEGY = "strategy";
   private static final String PROP_STRATEGY_DEFAULT = StrategyType.COMPOUND.toString();
+  private static final String PROP_EVENT_TYPE = "eventType";
 
   private static final long OVERFETCH = TimeUnit.DAYS.toMillis(2);
 
   private final StrategyType strategy;
   private final EventManager eventDAO;
   private final int k;
+  private final EventType eventType;
```

Comment:
Maybe stick to String here. EventType is more accurate, but not extensible to different event types people may add through external libraries

Reply:
Fixed